ui/src/components/song/SongHelpDialog.tsx

```diff
@@ -12,14 +12,14 @@ const SongHelpDialog: FC = () => {
         content={
           <>
             <Typography color="primary" mb={2}>
-              Use hashtags “#” to indicate sections <br />
+              Use hashtags “{`{ }`}” to indicate sections <br />
```

Comment:
this is not hashtags anymore. should be curly brackets? idk what's the official proper name

Reply:
ops forgot to change dat